architecture/service_catalog/index.adoc

```diff
@@ -101,6 +101,24 @@ references to the binding secret from the deployment. Otherwise, the next
 rollout will fail.
 ====
 
+[[service-catalog-route-planning]]
+== Service catalog compatibility with services outside of the cluster
+
+As of {product-title} version 3.10, a user can specify a Kubernetes service
+outside of the cluster to be consumed by other users using firewall rules. This
+is up to the user to ensure that the service is able to be consumed by adding
+rules. {product-title} ensures that the network isolation is automatically
+allowing connectivity to occur.
+
+Administrators can use `NetworkPolocy` objects to declare appropriate access to
+these resources from a user's project or namespace. For example, from the
+Ansible broker or the Template broker. These `NetworkPolocy` objects are
+triggered by using the service catalog through provisioning or binding.
+
+See the <here> for more information on how to trigger the service catalog.
```

Comment:
This is more about making sure that the service catalog installer remembers to put in any needed network policies to allow access to the service.